lib/teamlab/modules/projects/projects_discussions.rb

```diff
@@ -39,8 +39,22 @@ def subscribe_to_message_action(message_id)
       @request.put(['message', message_id.to_s, 'subscribe'])
     end
 
+    # Returns a list of all the subscribers of the discussion with the selected message
+    # @param message_id [Integer, String] Message ID
+    # @return [Array] List of subscibers
+    def get_discussion_subscribers(message_id)
```

Comment:
I think `rubocop` usually complain if methods starting with `get_`

I mostly agree and `discussion_subscribers` is a good name without `get_`